src/uptane_generator/repo.cc

```diff
@@ -39,6 +39,10 @@ void Repo::addDelegationToSnapshot(Json::Value *snapshot, const Uptane::Role &ro
   std::string signed_role = Utils::readFile(repo_dir / role_file_name);
 
   (*snapshot)["meta"][role_file_name]["version"] = role_json["version"].asUInt();
+  (*snapshot)["meta"][role_file_name]["length"] = signed_role.size();
+  (*snapshot)["meta"][role_file_name]["length"] = signed_role.size();
+  (*snapshot)["meta"][role_file_name]["hashes"]["sha256"] =
+      boost::algorithm::to_lower_copy(boost::algorithm::hex(Crypto::sha256digest(signed_role)));
```

Comment:
Just two questions:
1. why do we set length twice ?
2. how does it help with the fix ?

Reply:
1. just typo
2. it doesn't help with the fix, it's needed for the test.